### PR TITLE
Standardize code documentation in db/* - Closes #1471

### LINKED ISSUE #1471: Apply code documentation standards in db/*
Parent: #1333

### PR DESCRIPTION
**Pull request by Tschakki**
### What was the problem?
Code documentation standards are not applied consistently in the project.

### How did I fix it?
Apply the same JSDoc standards in every file.

### How to test it?
`npm run docs:build`
`npm run docs:serve`

### Review checklist

* The PR solves #1471 
* All new code is covered with unit tests
* All new code was formatted with Prettier
* Linting passes
* Tests pass
* Commit messages follow the [commit guidelines](CONTRIBUTING.md#git-commit-messages)
* Documentation has been added/updated

### CODE REVIEW

File: db/repos/blocks.js [hunk 1]
```diff
@@ -159,90 +177,109 @@ class BlocksRepository {
...
 	getBlocksForTransport(ids) {
 		// TODO: Must use a result-specific method, not .query
 		return this.db.query(sql.getBlocksForTransport, [ids]);
 	}
 
 	/**
-	 * Insert a block to database
+	 * Insert a block to database.
+	 *
+	 * @param {Object} block - Attributes to be inserted, can be any of [BlocksRepo's dbFields property]{@link BlocksRepo#dbFields}
```
Comment (willclarktech) [inline db/repos/blocks.js]: Looks like a line duplication.

File: db/repos/transactions.multisignature.js [hunk 2]
```diff
@@ -49,9 +52,11 @@ function MultiSigTransactionsRepo(db, pgp) {
 }
 
 /**
- * Save Multisignature transactions
+ * Save Multisignature transactions.
+ *
  * @param {Array.<{id: string, {asset: {multisignature: {min: int, lifetime: int, keysgroup: Array.<string>}}}>} transactions
```
Comment (willclarktech) [inline db/repos/transactions.multisignature.js]: This typedef seems to break GitHub parsing. Is it valid?

File: docs/conf.json [hunk 3]
```diff
@@ -8,6 +8,7 @@
       "./docs/typedef/",
       "./app.js",
       "./api/",
+	  "./db/",
```
Comment (willclarktech) [inline docs/conf.json]: Indentation

File: db/repos/accounts.js [hunk 4]
```diff
@@ -106,7 +109,8 @@ class AccountsRepository {
 	/**
 	 * Get list of all database fields.
 	 *
-	 * @return {array}
+	 * @returns {array}
```
[the remark this answers is not in the document]
Comment (Tschakki) [reply db/repos/accounts.js]: The `@function`-tag is only needed if JSDoc does not recognize a function by itself from the code.

File: db/repos/transactions.multisignature.js (hunk 2 above)
Comment (Tschakki) [reply db/repos/transactions.multisignature.js]: I found no example in the docs.
I simplified it by putting only {Array} as a type.
For the future we could use typedefs like recommended in the JSDoc documentation for describing more complex Objects and Arrays.

File: db/index.js [hunk 5]
```diff
@@ -37,13 +43,21 @@ let initOptions = {
...
+/**
+ * Connects to the database.
+ *
  * @function connect
- * @param {Object} config
- * @param {function} logger
- * @return {Promise<pg-promise.Database>}
+ * @param {Object} config - Description of the param
+ * @param {function} logger - Description of the param
+ * @return {Promise}
```
Comment (willclarktech) [inline db/index.js]: Should these have `@todos`?

**Pull request merged**